modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-creating-at-once_{context}"]
+= Creating AWS resources with a single command
+
+// Would be nice to have a little explanation why a user would want to go this route vs. the other option.
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--target-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. To place JSON files on the local file system instead, use the `--generate-files-only` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* Extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Extract the list of `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=./credrequests quay.io/__<path_to>__/openshift-release:__<version>__
+----
+
+. Process every `CredentialsRequest` object in the `./credrequests` directory with `ccoctl`:
+// My assumption is that it does all `CredentialsRequest` objects rather than "each" as in the other procedure.
++
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl aws create-all --name-prefix=__<name_prefix>__ --region=__<aws_region>__ --credentials-requests-dir=./credrequests
```

Comment:
Digging around in Jira this afternoon I see s/name-prefix/name, will update throughout

Reply:
Related Q: what does "tag and account any cloud resources" mean? Specifically the "account" part.

Reply:
Perhaps it would be better worded as "tag for tracking". Basically, the tool names things using the `--name` flag, and tags them using that field as well.